test/specs/commands/createTransactionRegisterSecondPassphrase.js

```diff
@@ -0,0 +1,129 @@
+								it('Then it should reject with message "Unknown second passphrase source type. Must be one of `file`, or `stdin`."', then.itShouldRejectWithMessage);
+							});
+						});
+						describe('Given an empty options object', () => {
+							beforeEach(given.anEmptyOptionsObject);
+							describe('Given the passphrase and the second passphrase are provided via the prompt', () => {
+								beforeEach(given.thePassphraseAndTheSecondPassphraseAreProvidedViaThePrompt);
+								describe('When the action is called with the options', () => {
+									beforeEach(when.theActionIsCalledWithTheOptions);
+									it('Then it should not get the passphrase from stdin', then.itShouldNotGetThePassphraseFromStdIn);
+									it('Then it should get the passphrase using the vorpal instance', then.itShouldGetThePassphraseUsingTheVorpalInstance);
+									it('Then it should get the passphrase with a repeated prompt', then.itShouldGetThePassphraseWithARepeatedPrompt);
+									it('Then it should get the second passphrase using the vorpal instance', then.itShouldGetTheSecondPassphraseUsingTheVorpalInstance);
+									it('Then it should get the second passphrase with a repeated prompt', then.itShouldGetTheSecondPassphraseWithARepeatedPrompt);
+									it('Then it should have created a register second passphrase transaction using the passphrase and the second passphrase', then.itShouldHaveCreatedARegisterSecondPassphraseTransactionUsingThePassphraseAndTheSecondPassphrase);
+									it('Then it should resolve to the created transaction', then.itShouldResolveToTheCreatedTransaction);
+								});
+							});
+						});
+						describe('Given an options object with passphrase set to "stdin"', () => {
+							beforeEach(given.anOptionsObjectWithPassphraseSetTo);
+							describe('Given the passphrase is provided via stdin', () => {
+								beforeEach(given.thePassphraseIsProvidedViaStdIn);
+								describe('Given the second passphrase is provided via the prompt', () => {
+									beforeEach(given.theSecondPassphraseIsProvidedViaThePrompt);
+									describe('When the action is called with the options', () => {
+										beforeEach(when.theActionIsCalledWithTheOptions);
+										it('Then it should get the passphrase from stdin', then.itShouldGetThePassphraseFromStdIn);
+										it('Then it should get the passphrase using the passphrase from stdin', then.itShouldGetThePassphraseUsingThePassphraseFromStdIn);
+										it('Then it should get the second passphrase using the vorpal instance', then.itShouldGetTheSecondPassphraseUsingTheVorpalInstance);
+										it('Then it should get the second passphrase with a repeated prompt', then.itShouldGetTheSecondPassphraseWithARepeatedPrompt);
+										it('Then it should create a register second passphrase transaction using the passphrase and the second passphrase', then.itShouldHaveCreatedARegisterSecondPassphraseTransactionUsingThePassphraseAndTheSecondPassphrase);
```

Comment:
I prefer this description to the `should have created` version, but the function name diverges currently.